sw/device/lib/pinmux.c

```diff
@@ -11,19 +11,32 @@
 
 #define PINMUX0_BASE_ADDR TOP_EARLGREY_PINMUX_AON_BASE_ADDR
 
+// TODO: autogenerated constants for this will be available once
```

Comment:
@silvestrst @GregAC 

Since the UART is now a muxed peripheral, I had to hardcode the corresponding pinmux configuration here to get our simulation and FPGA emulation environments up and running again.

This is not pretty, and intended to be a temporary workaround until the pinmux/padctrl/pinout changes went in and until the DIF is available.

Let me know if you think this should be done differently.

Reply:
@msfschaffner thank you, looks good to me!

I think eventually the plan is to remove this file, and to the configuration directly in the bootrom using `dif_pinmux`.